.circleci/config.yml

```diff
@@ -116,6 +134,10 @@ workflows:
         - check-style
         - test-sqlite
         - test-postgres
+    - scan-image:
+        context: matterbuild-docker
+        requires:
+        - push-docker-pr
```

Comment:
Do we want to scan the image before pushing it? Not sure we would want to start blocking pushes right now if the scan failed, but also not sure what we have in mind for scan automation.

Reply:
we need to push because the scanner fetches from the registry. however i can see if that works without pushing